apps/dashboard/views.py

```diff
@@ -52,9 +53,14 @@ def dashboard_view(request: HttpRequest) -> HttpResponse:
     """
     from apps.tasks.tasks import TASK_FUNCTIONS
 
+    prefix = os.getenv("METRICS_URL_PREFIX", "None")
```

Comment:
```suggestion
    prefix = os.getenv("METRICS_SERVICE_URL_PREFIX", None)
```

(add SERVICE, remove quotes around None)

(see also https://github.com/ansible/metrics-service/pull/69#discussion_r2603553411)
EDIT: created AAP-60451 ; but probably resolved in #70